### PR TITLE
Mentor evaluation

### PR DESCRIPTION
**Pull request by RMaksymczuk**
It is almost finished, with backend and fontend. The thing that is still missing, is 'soundness' value, that I'm not sure if we really need to implement. Another one, is a real time score / low score warning. I'll finish that in meantime.

There is a file in 'config/mentor_evaluation.yml' which contains all the questions and possible answers, with scores. It would be good if the person who will be responsible for evaluation could improve it and correct mistakes.

The backend I made is partly prepared for mentee evaluation. It could be done similarly. 

I'm open for any advice about the code :)

### CODE REVIEW

File: app/assets/javascripts/application/evaluations.js.coffee [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#= require 'select2.min'
```
Comment (DMips) [inline app/assets/javascripts/application/evaluations.js.coffee]: can you create class here with methods?

File: db/migrate/20151206185429_create_evaluations.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class CreateEvaluations < ActiveRecord::Migration
+  def change
+    create_table :evaluations do |t|
+      t.float :score
```
Comment (DMips) [inline db/migrate/20151206185429_create_evaluations.rb]: can we use decimal here? float uses sometimes strange math ;)

Comment (RMaksymczuk) [inline db/migrate/20151206185429_create_evaluations.rb]: good point, I'll do that :)

File: app/assets/javascripts/application/evaluations.js.coffee (hunk 1 above)
Comment (RMaksymczuk) [inline app/assets/javascripts/application/evaluations.js.coffee]: I'm not working with javascript classes, mostly. But it's a good idea to use it here. I'll update it today's evening.

File: app/controllers/evaluations_controller.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+class EvaluationsController < ApplicationController
```
Comment (DMips) [inline app/controllers/evaluations_controller.rb]: please add before_action for checking whether current_user is organiser or not (there's helper for that in application_controller)

**Pull request merged**